udata_search_service/infrastructure/search_clients.py

```diff
@@ -339,6 +359,54 @@ def query_reuses(self, query_text: str, offset: int, page_size: int, filters: di
         res = [hit.to_dict(skip_empty=False) for hit in response.hits]
         return results_number, res
 
+    def query_dataservices(self, query_text: str, offset: int, page_size: int, filters: dict, sort: Optional[str] = None) -> Tuple[int, List[dict]]:
+        s = SearchableDataservice.search()
+
+        for key, value in filters.items():
+            s = s.filter('term', **{key: value})
+
+        dataservices_score_functions = [
```

Comment:
We can update the score functions factors depending on search results in production